app/js/filters.js

```diff
@@ -203,25 +201,24 @@ angular.module('myApp.filters', ['myApp.i18n'])
       var totalParts = total.split(' ')
 
       if (totalParts[1] === doneParts[1]) {
-        return _('format_size_progress_mulitple', {done: doneParts[0], total: totalParts[0], parts: (doneParts[1] || '')})
+        return _('format_size_progress_mulitple', {done: doneParts[0], total: totalParts[0], parts: doneParts[1]})
```

Comment:
I removed the "if" because it was unreachable code. 
donePart[1] would only be undefined if totalPart[1] is undefined